sycl/doc/extensions/proposed/sycl_ext_oneapi_kernel_arg_restrict_property.asciidoc

```diff
@@ -0,0 +1,189 @@
+= sycl_ext_oneapi_kernel_arg_properties
```

Comment:
You should also rename the file to match "sycl/doc/extensions/proposed/sycl_ext_oneapi_kernel_arg_properties.asciidoc"

Reply:
Renamed.